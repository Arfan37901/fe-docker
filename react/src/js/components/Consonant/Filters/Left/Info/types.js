import {
    bool,
    node,
    func,
    shape,
    number,
    arrayOf,
} from 'prop-types';

import { SortOptionType } from '../../../types/config';

// eslint-disable-next-line import/prefer-default-export
export const InfoType = {
    cardsQty: number,
    filtersQty: number,
    windowWidth: number,
    enabled: bool.isRequired,
    selectedFiltersQty: number,
    sortComponent: node.isRequired,
    searchComponent: node.isRequired,
    onMobileFiltersToggleClick: func.isRequired,
    sortOptions: arrayOf(shape(SortOptionType)).isRequired,
};
