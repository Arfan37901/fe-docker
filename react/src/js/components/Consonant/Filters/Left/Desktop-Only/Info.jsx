import PropTypes from 'prop-types';
import React from 'react';
import classNames from 'classnames';

const Info = (props) => {
    const {
        title,
        totalResultsText,
        sortEnabled,
        sortOptions,
        showTotalResults,
    } = props;

    const wrapperClassName = classNames({
        'consonant-filters-info--wrapper': true,
        'consonant-filters-info--wrapper_no-line': !sortEnabled || !sortOptions.length,
    });

    return (
        <div className={wrapperClassName}>
            {title &&
                <h2
                    data-testid="title"
                    className="consonant-filters-info--title">
                    {title}
                </h2>
            }
            {showTotalResults &&
                <span
                    data-testid="results"
                    className="consonant-filters-info--results">
                    {totalResultsText}
                </span>
            }
        </div>
    );
};

/* eslint-disable-next-line import/prefer-default-export */
export { Info };

Info.propTypes = {
    title: PropTypes.string,
    totalResultsText: PropTypes.string,
    sortEnabled: PropTypes.bool,
    sortOptions: PropTypes.arrayOf(PropTypes.object).isRequired,
    showTotalResults: PropTypes.bool,
};

Info.defaultProps = {
    title: '',
    totalResultsText: '',
    sortEnabled: false,
    showTotalResults: false,
};
