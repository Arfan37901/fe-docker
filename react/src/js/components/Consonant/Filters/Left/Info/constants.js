
/**
 * Minimal viewport width to fit desktops/laptops
 * @type {Number}
 */
export const DESKTOP_MIN_WIDTH = 1200;

export const defaultProps = {
    cardsQty: 0,
    filtersQty: 0,
    selectedFiltersQty: 0,
    windowWidth: window.innerWidth,
};
