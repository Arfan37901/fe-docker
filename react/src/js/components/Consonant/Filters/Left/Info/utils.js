// eslint-disable-next-line import/prefer-default-export
export const selector = ({
    sort: { enabled: sortEnabled },
    search: { enabled: searchEnabled },
    filterPanel: {
        enabled: filterPanelEnabled,
        i18n: {
            leftPanel: {
                mobile: { filtersBtnLabel },
            },
        },
    },
    collection: {
        showTotalResults,
        i18n: { title, totalResultsText },
    },
}) => ({
    title,
    sortEnabled,
    searchEnabled,
    showTotalResults,
    totalResultsText,
    filterPanelEnabled,
    mobileFilterButtonlabel: filtersBtnLabel,
});
