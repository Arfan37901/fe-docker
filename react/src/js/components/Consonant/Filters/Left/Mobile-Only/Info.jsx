import React from 'react';
import classNames from 'classnames';
import {
    string,
    number,
    func,
} from 'prop-types';

const infoType = {
    selectedFiltersQty: number,
    mobileFilterBtnLabel: string,
    onMobileFiltersToggleClick: func.isRequired,
};

const defaultProps = {
    selectedFiltersQty: 0,
    mobileFilterBtnLabel: '',
};

/**
 * Button with the count of selected filters
 * for mobile and tablet breakpoints
 *
 * @component
 * @example
 * const props= {
    onMobileFiltersToggleClick: Function,
    mobileFilterBtnLabel: String,
    selectedFiltersQty: Number,
 * }
 * return (
 *   <Info {...props}/>
 * )
 */
const Info = ({
    selectedFiltersQty,
    mobileFilterBtnLabel,
    onMobileFiltersToggleClick,
}) => {
    const atleastOneSelectedFilter = selectedFiltersQty > 0;

    /**
     * Class name for the button:
     * whether the button should display the quantity of the selected filters or not
     * @type {String}
     */
    const selectedFiltersQtyClassName = classNames({
        'consonant-FiltersInfo-btn': true,
        'consonant-FiltersInfo-btn--withFilters': atleastOneSelectedFilter,
    });

    return (
        <div
            data-testid="btn-wrapper"
            className="consonant-FiltersInfo-btnWrapper">
            <button
                type="button"
                data-testid="info-btn"
                className={selectedFiltersQtyClassName}
                onClick={onMobileFiltersToggleClick}>
                <span
                    className="consonant-FiltersInfo-btnIco" />
                <span
                    className="consonant-FiltersInfo-btnText">
                    {mobileFilterBtnLabel}
                </span>
                {atleastOneSelectedFilter &&
                    <span
                        data-testid="btn-selected"
                        className="consonant-FiltersInfo-btnSelected">
                        {selectedFiltersQty}
                    </span>
                }
            </button>
        </div>
    );
};

Info.propTypes = infoType;
Info.defaultProps = defaultProps;

/* eslint-disable-next-line import/prefer-default-export */
export { Info };
