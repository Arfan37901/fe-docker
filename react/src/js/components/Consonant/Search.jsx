import React from 'react';
import PropTypes from 'prop-types';

const Search = (props) => {
    const { itemsQty, value, onSearch } = props;
    const handleSearch = (evt) => {
        onSearch(evt.target.value);
    };

    const clearSearch = () => {
        onSearch('');
    };

    return (
        <div className="consonant-search">
            <label>
                <span className="consonant-search--input-title">Search</span>
                <span className="consonant-search--input-wrapper">
                    <input
                        type="search"
                        placeholder={onSearch ? `Search within ${itemsQty} items` : 'Search'}
                        value={value}
                        onChange={handleSearch}
                        required />
                    <button
                        type="button"
                        title="Click to clear search query"
                        className="consonant-search--input-clear"
                        onClick={clearSearch}>clear
                    </button>
                </span>
            </label>
        </div>
    );
};

export default Search;

Search.propTypes = {
    itemsQty: PropTypes.number,
    onSearch: PropTypes.func.isRequired,
    value: PropTypes.string.isRequired,
};

Search.defaultProps = {
    itemsQty: 0,
};
