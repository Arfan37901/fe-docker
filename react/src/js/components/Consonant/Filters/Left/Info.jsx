import React from 'react';
import PropTypes from 'prop-types';
import classNames from 'classnames';

import { Info as MobileInfo } from './Mobile-Only/Info';
import { renderTotalResults } from '../../Helpers/rendering';

import { useConfig } from '../../Helpers/hooks';

/**
 * Left filter info panel
 *
 * @component
 * @example
 * const props= {
    enabled: Boolean,
    filtersQty: Number,
    cardsQty: Number,
    selectedFiltersQty: Number,
    onMobileFiltersToggleClick: Function,
    searchComponent: Node,
    sortComponent: Node,
    sortOptions: Array,
    windowWidth: Number,
 * }
 * return (
 *   <Info {...props}/>
 * )
 */
const Info = (props) => {
    const {
        enabled,
        filtersQty,
        cardsQty,
        selectedFiltersQty,
        onMobileFiltersToggleClick,
        searchComponent,
        sortComponent,
        sortOptions,
        windowWidth,
    } = props;

    const getConfig = useConfig();

    /**
     **** Authored Configs ****
     */
    const title = getConfig('collection', 'i18n.title');
    const enableFilterPanel = getConfig('filterPanel', 'enabled');
    const showTotalResults = getConfig('collection', 'showTotalResults');
    const showTotalResultsText = getConfig('collection', 'i18n.totalResultsText');
    const searchEnabled = getConfig('search', 'enabled');
    const sortEnabled = getConfig('sort', 'enabled');
    const mobileFilterBtnLabel = getConfig('filterPanel', 'i18n.leftPanel.mobile.filtersBtnLabel');

    /**
     **** Constants ****
     */

    /**
     * Total results HTML
     * @type {Array}
     */
    const totalResultsHtml = renderTotalResults(showTotalResultsText, cardsQty);

    /**
     * Minimal viewport width to fit desktops/laptops
     * @type {Number}
     */
    const DESKTOP_MIN_WIDTH = 1200;

    /**
     * Whether the current viewport width fits tablets or mobile devices
     * @type {Boolean}
     */
    const NOT_DESKTOP_SCREEN_SIZE = windowWidth < DESKTOP_MIN_WIDTH;

    /**
     * Whether the sort dropdown should be displayed
     * @type {Boolean}
     */
    const shouldRenderSortPopup = sortEnabled && sortOptions.length > 0;

    /**
     * Whether the search bar should be displayed
     * @type {Boolean}
     */
    const shouldRenderSearch = searchEnabled && NOT_DESKTOP_SCREEN_SIZE;

    /**
     * Whether the left filters info block should be displayed
     * on mobile and tablet breakpoints
     * @type {Boolean}
     */
    const shouldRenderMobileInfo = NOT_DESKTOP_SCREEN_SIZE && filtersQty > 0 && enabled;

    /**
     * Class name for the left filters info:
     * whether the left filters info block should share width with left filters panel
     * @type {String}
     */
    const containerClassName = classNames({
        'consonant-filters-info': true,
        'consonant-filters-info_no-filter-panel': !enableFilterPanel,
    });

    /**
     * Class name for the left filters info inner wrapper:
     * whether the left filters info inner wrapper should display a vertical separator
     * after the block with the quantity of the filtered cards
     * @type {String}
     */
    const wrapperClassName = classNames({
        'consonant-filters-info--wrapper': true,
        'consonant-filters-info--wrapper_no-line': !sortEnabled || !sortOptions.length,
    });

    return (
        <aside
            data-testid="consonant-filters__info"
            className={containerClassName}>
            <div
                className="consonant-filters-info--search">
                {shouldRenderSearch && searchComponent}
            </div>
            {shouldRenderMobileInfo &&
                <MobileInfo
                    selectedFiltersQty={selectedFiltersQty}
                    mobileFilterBtnLabel={mobileFilterBtnLabel}
                    onMobileFiltersToggleClick={onMobileFiltersToggleClick} />
            }
            {shouldRenderSortPopup && sortComponent}
            <div
                className={wrapperClassName}>
                {title &&
                    <h2
                        data-testid="title"
                        className="consonant-filters-info--title">
                        {title}
                    </h2>
                }
                {showTotalResults &&
                    <div
                        data-testid="results"
                        className="consonant-filters-info--results">
                        {totalResultsHtml}
                    </div>
                }
            </div>
        </aside>
    );
};

Info.propTypes = {
    enabled: PropTypes.bool.isRequired,
    filtersQty: PropTypes.number,
    cardsQty: PropTypes.number,
    selectedFiltersQty: PropTypes.number,
    onMobileFiltersToggleClick: PropTypes.func.isRequired,
    searchComponent: PropTypes.node.isRequired,
    sortComponent: PropTypes.node.isRequired,
    sortOptions: PropTypes.arrayOf(PropTypes.object).isRequired,
    windowWidth: PropTypes.number,
};

Info.defaultProps = {
    filtersQty: 0,
    cardsQty: 0,
    selectedFiltersQty: 0,
    windowWidth: window.innerWidth,
};

export { Info };
