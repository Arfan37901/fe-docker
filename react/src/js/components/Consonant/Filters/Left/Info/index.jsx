import React from 'react';
import classNames from 'classnames';

import { selector } from './utils';
import { InfoType } from './types';
import { If } from '../../../Common';
import { Info as MobileInfo } from '../Mobile-Only/Info';
import { useConfigSelector } from '../../../Helpers/hooks';
import { DESKTOP_MIN_WIDTH, defaultProps } from './constants';
import { renderTotalResults } from '../../../Helpers/rendering';

/**
 * Left filter info panel
 *
 * @component
 * @example
 * const props= {
    enabled: Boolean,
    filtersQty: Number,
    cardsQty: Number,
    selectedFiltersQty: Number,
    onMobileFiltersToggleClick: Function,
    searchComponent: Node,
    sortComponent: Node,
    sortOptions: Array,
    windowWidth: Number,
 * }
 * return (
 *   <Info {...props}/>
 * )
 */
const Info = ({
    enabled,
    cardsQty,
    filtersQty,
    sortOptions,
    windowWidth,
    sortComponent,
    searchComponent,
    selectedFiltersQty,
    onMobileFiltersToggleClick,
}) => {
    const {
        title,
        sortEnabled,
        searchEnabled,
        showTotalResults,
        totalResultsText,
        filterPanelEnabled,
        mobileFilterButtonlabel,
    } = useConfigSelector(selector);

    /**
     **** Authored Configs ****
     */

    /**
     **** Constants ****
     */

    /**
     * Total results HTML
     * @type {Array}
     */
    const totalResultsHtml = renderTotalResults(totalResultsText, cardsQty);

    /**
     * Whether the current viewport width fits tablets or mobile devices
     * @type {Boolean}
     */
    const notDesktopScreenSize = windowWidth < DESKTOP_MIN_WIDTH;

    /**
     * Whether the sort dropdown should be displayed
     * @type {Boolean}
     */
    const shouldRenderSortPopup = sortEnabled && sortOptions.length > 0;

    /**
     * Whether the search bar should be displayed
     * @type {Boolean}
     */
    const shouldRenderSearch = searchEnabled && notDesktopScreenSize;

    /**
     * Whether the left filters info block should be displayed
     * on mobile and tablet breakpoints
     * @type {Boolean}
     */
    const shouldRenderMobileInfo = notDesktopScreenSize && filtersQty > 0 && enabled;

    /**
     * Class name for the left filters info:
     * whether the left filters info block should share width with left filters panel
     * @type {String}
     */
    const containerClassName = classNames({
        'consonant-filters-info': true,
        'consonant-filters-info_no-filter-panel': !filterPanelEnabled,
    });

    /**
     * Class name for the left filters info inner wrapper:
     * whether the left filters info inner wrapper should display a vertical separator
     * after the block with the quantity of the filtered cards
     * @type {String}
     */
    const wrapperClassName = classNames({
        'consonant-filters-info--wrapper': true,
        'consonant-filters-info--wrapper_no-line': !sortEnabled || !sortOptions.length,
    });

    return (
        <aside
            data-testid="consonant-filters__info"
            className={containerClassName}>
            <div
                className="consonant-filters-info--search">
                <If condition={Boolean(shouldRenderSearch)}>
                    {searchComponent}
                </If>
            </div>
            <If condition={Boolean(shouldRenderMobileInfo)}>
                <MobileInfo
                    selectedFiltersQty={selectedFiltersQty}
                    mobileFilterBtnLabel={mobileFilterButtonlabel}
                    onMobileFiltersToggleClick={onMobileFiltersToggleClick} />
            </If>
            <If condition={Boolean(shouldRenderSortPopup)}>
                {sortComponent}
            </If>
            <div
                className={wrapperClassName}>
                <If condition={Boolean(title)}>
                    <h2
                        data-testid="title"
                        className="consonant-filters-info--title">
                        {title}
                    </h2>
                </If>
                <If condition={Boolean(showTotalResults)}>
                    <div
                        data-testid="results"
                        className="consonant-filters-info--results">
                        {totalResultsHtml}
                    </div>
                </If>
            </div>
        </aside>
    );
};

Info.propTypes = InfoType;
Info.defaultProps = defaultProps;

/* eslint-disable-next-line import/prefer-default-export */
export { Info };
