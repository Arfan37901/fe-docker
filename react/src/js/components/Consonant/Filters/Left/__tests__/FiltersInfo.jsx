import React from 'react';
import { screen, fireEvent } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

import Info from '../FilterInfo';
import Search from '../../../Search/Search';
import Select from '../../../Select/Select';

import {
    DEFAULT_PROPS,
    NON_DESKTOP_WIDTH,
    selectedAllFilters,
} from '../../../Helpers/Testing/Constants/FiltersInfo';
import { DEFAULT_PROPS as SELECT_DEFAULT_PROPS } from '../../../Helpers/Testing/Constants/Select';
import { DEFAULT_PROPS as SEARCH_DEFAULT_PROPS } from '../../../Helpers/Testing/Constants/Search';

import makeSetup, { createTree } from '../../../Helpers/Testing/Utils/Settings';

const setup = makeSetup(Info, DEFAULT_PROPS);

describe('Consonant/FiltersInfo', () => {
    test('should render without title', () => {
        setup({}, { collection: { i18n: { title: '' } } });

        const titleElement = screen.queryByTestId('title');

        expect(titleElement).toBeNull();
    });
    test('should render without total result', () => {
        setup({}, { collection: { showTotalResults: false } });

        const resultsElement = screen.queryByTestId('results');

        expect(resultsElement).toBeNull();
    });
    test('should render all list items', () => {
        const {
            props: { filters },
        } = setup({ selectedFiltersQty: 1, filters: selectedAllFilters });

        const filterItemsLength = filters.reduce((acc, filter) =>
            acc + filter.items.length, 0);

        const selectedFiltersWrapper = screen.queryByTestId('selected-filters');

        expect(selectedFiltersWrapper).not.toBeNull();

        const selectedFilterList = screen.queryAllByTestId('selected-filter');

        expect(selectedFilterList).toHaveLength(filterItemsLength);
    });
    test('shouldn`t render list items', () => {
        setup();

        const selectedFiltersWrapper = screen.queryByTestId('selected-filters');

        expect(selectedFiltersWrapper).toBeNull();
    });
    test('should render mobile button', () => {
        setup({ windowWidth: NON_DESKTOP_WIDTH, selectedFiltersQty: 1 });

        const btnWrapperElement = screen.queryByTestId('btn-wrapper');

        expect(btnWrapperElement).not.toBeNull();

        const btnSelectedElement = screen.queryByTestId('btn-selected');

        expect(btnSelectedElement).not.toBeNull();
        expect(btnSelectedElement).toHaveTextContent('1');
    });

    describe('Check snapshots', () => {
        test('should renders correctly with two children', () => {
            const tree = createTree((
                <Info {...DEFAULT_PROPS} >
                    <Search
                        name="searchFiltersInfo"
                        {...SEARCH_DEFAULT_PROPS} />
                    <Select
                        childrenKey="selectFiltersInfo"
                        {...SELECT_DEFAULT_PROPS} />
                </Info>
            ));

            expect(tree).toMatchSnapshot();
        });
        test('should renders correctly with search', () => {
            const tree = createTree((
                <Info {...DEFAULT_PROPS} >
                    <Search
                        name="searchFiltersInfo"
                        {...SEARCH_DEFAULT_PROPS} />
                </Info>
            ));

            expect(tree).toMatchSnapshot();
        });
        test('should renders correctly with select', () => {
            const tree = createTree((
                <Info {...DEFAULT_PROPS} >
                    <Select
                        childrenKey="selectFiltersInfo"
                        {...SELECT_DEFAULT_PROPS} />
                </Info>
            ));

            expect(tree).toMatchSnapshot();
        });
        test('should renders correctly when desktop width', () => {
            const { tree } = setup();

            expect(tree).toMatchSnapshot();
        });
        test('should renders correctly when mobile width', () => {
            const { tree } = setup({ windowWidth: NON_DESKTOP_WIDTH });

            expect(tree).toMatchSnapshot();
        });
    });

    describe('Interaction with UI', () => {
        test('should call onSelectedFilterClick', () => {
            const { props: { onSelectedFilterClick } } = setup({
                filters: selectedAllFilters,
                selectedFiltersQty: 1,
            });

            const [selectedFilter] = screen.queryAllByTestId('selected-filter');

            expect(selectedFilter).toBeDefined();

            fireEvent.click(selectedFilter);

            expect(onSelectedFilterClick).toBeCalled();
        });
        test('should call onMobileFiltersToggleClick', () => {
            const { props: { onMobileFiltersToggleClick } } = setup({
                windowWidth: NON_DESKTOP_WIDTH,
            });

            const buttonElement = screen.queryByTestId('info-btn');

            expect(buttonElement).not.toBeNull();

            fireEvent.click(buttonElement);

            expect(onMobileFiltersToggleClick).toBeCalled();
        });
    });
});
