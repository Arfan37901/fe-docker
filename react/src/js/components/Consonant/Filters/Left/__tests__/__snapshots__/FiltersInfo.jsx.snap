// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Consonant/FiltersInfo Check snapshots should renders correctly when desktop width 1`] = `
<aside
  className="consonant-filters-info"
  data-testid="consonant-filters__info"
>
  <div
    className="consonant-filters-info--wrapper consonant-filters-info--wrapper_no-line"
  >
    
    <span
      className="consonant-filters-info--results"
      data-testid="results"
    >
      0 results
    </span>
  </div>
  <div
    className="consonant-filters-info--search"
  />
</aside>
`;

exports[`Consonant/FiltersInfo Check snapshots should renders correctly when mobile width 1`] = `
<aside
  className="consonant-filters-info"
  data-testid="consonant-filters__info"
>
  <div
    className="consonant-filters-info--search"
  />
  <div
    className="consonant-filters-info--btn-wrapper"
    data-testid="btn-wrapper"
  >
    <button
      className="consonant-filters-info--btn"
      data-testid="info-btn"
      onClick={[MockFunction]}
      type="button"
    >
      <span
        className="consonant-filters-info--btn-ico"
      />
      <span
        className="consonant-filters-info--btn-text"
      >
        filters
      </span>
    </button>
  </div>
</aside>
`;

exports[`Consonant/FiltersInfo Check snapshots should renders correctly with search 1`] = `
<aside
  className="consonant-filters-info"
  data-testid="consonant-filters__info"
>
  <div
    className="consonant-filters-info--wrapper consonant-filters-info--wrapper_no-line"
  >
    
    <span
      className="consonant-filters-info--results"
      data-testid="results"
    >
      0 results
    </span>
  </div>
  <div
    className="consonant-filters-info--search"
  />
</aside>
`;

exports[`Consonant/FiltersInfo Check snapshots should renders correctly with select 1`] = `
<aside
  className="consonant-filters-info"
  data-testid="consonant-filters__info"
>
  <div
    className="consonant-filters-info--wrapper consonant-filters-info--wrapper_no-line"
  >
    
    <span
      className="consonant-filters-info--results"
      data-testid="results"
    >
      0 results
    </span>
  </div>
  <div
    className="consonant-filters-info--search"
  />
</aside>
`;

exports[`Consonant/FiltersInfo Check snapshots should renders correctly with two children 1`] = `
<aside
  className="consonant-filters-info"
  data-testid="consonant-filters__info"
>
  <div
    className="consonant-filters-info--wrapper consonant-filters-info--wrapper_no-line"
  >
    
    <span
      className="consonant-filters-info--results"
      data-testid="results"
    >
      0 results
    </span>
  </div>
  <div
    className="consonant-filters-info--search"
  />
</aside>
`;
